sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -94,3 +94,52 @@ def _can_handle_query(cls, *query):
             if x.__class__.__name__ == 'Instrument' and x.value == 'noaa-predict':
                 return all(chklist)
         return False
+
+class SRSClient(GenericClient):
+
+    @staticmethod
+    def _get_default_uri():
+        today = datetime.datetime.utcnow()
+        return ['ftp://ftp.swpc.noaa.gov/pub/warehouse/{date:%Y}/SRS/{date:%Y%m%d}SRS.txt'.format(date=today)]
+
+    def _get_url_for_timerange(self, timerange, **kwargs):
+
+        if not timerange:
+            return SRSClient._get_default_uri()
+        result = list()
+        base_url = 'ftp://ftp.swpc.noaa.gov/pub/warehouse/'
+        total_days = (timerange.end - timerange.start).days + 1
+        all_dates = timerange.split(total_days)
+        for day in all_dates:
+            suffix = '{date:%Y}/SRS/{date:%Y%m%d}SRS.txt'
+            url = base_url + suffix.format(date=day.end)
+            result.append(url)
+        return result
+
+    def _makeimap(self):
+        self.map_['source'] = 'swpc'
+        self.map_['instrument'] = 'SOON'
+        self.map_['physobs'] = 'SRS'
+        self.map_['source'] = 'NOAA/USAF'
+
+    @classmethod
+    def _can_handle_query(cls, *query):
+        """
+        Answers whether client can service the query.
+
+        Parameters
+        ----------
+        query : list of query objects
+
+        Returns
+        -------
+        boolean
+            answer as to whether client can service the query
+        """
+        chkattr = ["Time", "Instrument"]
+        chklist = [x.__class__.__name__ in chkattr for x in query]
+        for x in query:
+            if x.__class__.__name__ == "Instrument" and x.value == "SOON":
```

Comment:
Yes, the telescope used to observe these AR is the [Solar Optical Observing Network](http://www.swpc.noaa.gov/products/solar-region-summary) (details tab)